packages/app/src/app/overmind/effects/router.ts

```diff
@@ -3,7 +3,7 @@ import { getSandboxOptions } from '@codesandbox/common/lib/url';
 import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
 import history from '../../utils/history';
 
-export default {
+export default new (class RouterEffect {
```

Comment:
Any reason why you created a class that's automatically invoked?
Isn't that just the same as an object, since you're not using any `this`? 🤔

Reply:
Yes, I'm using `this` here now!